accessibility-oscilloscope/package/sigproc/sigproc.mk

```diff
@@ -4,12 +4,15 @@
 #
 ################################################################################
 
-SIGPROC_VERSION = stub-v0.1
+SIGPROC_VERSION = v0.2
```

Comment:
no longer packaging the stub

Reply:
Wait let me make a new tag for the latest for that

Reply:
v0.3 now